platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanTest.java

```diff
@@ -206,6 +260,12 @@ public void testDestroy() throws Exception {
     public void testDestroyException() throws Exception {
         ApplicationServiceBean serviceBean = new ApplicationServiceBean(testAppService,
                 testConfigAdminExt);
+
+        serviceBean.setMBeanServer(mBeanServer);
+
+        doThrow(new NullPointerException()).when(mBeanServer)
+                .unregisterMBean(any(ObjectName.class));
```

Comment:
Since `unregisterMBean() can throw two checked exceptions, i.e., `InstanceNotFoundException` and `MBeanRegistrationException`, we should have two more test methods to test for those scenarios.